lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -389,6 +396,12 @@ def self.get_values(json, category_type, override_price = false)
       'i2.4xlarge' => [1890, 172192],
       'i2.8xlarge' => [3618, 313304],
       'i2.xlarge' => [480, 45439],
+      'i3.16xlarge' => [1351600000, 3300000],	# NVMe SSD: 3.3M iops
+      'i3.8xlarge'  => [ 675800000, 1650000],
+      'i3.4xlarge'  => [ 337900000,  825000],
+      'i3.2xlarge'  => [ 168950000,  412500],
+      'i3.xlarge'   => [  84475000,  206250],
+      'i3.large'    => [  42237500,  103125],
```

Comment:
the i3.16xlarge is quoted (with 8 NVMe SSD) at 3.3M iops at 4KB blocks, and each successive smaller instance halves the number of NVMe SSD's.